lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -94,6 +94,9 @@ do {
   $err_cur_line = SlurpOutput $err_file $err_cur_line 'err'
 } while (!($registered_task.state -eq 3))
 
+del $out_file
```

Comment:
@mwrock they won't get yanked in the beginning if they are uniquely named based on the process id because the next time through it would be a new process.  In our use case we could have 1000 requests on the same connection - each creating a unique log file - meaning *lots* of cruft out there in the temp directory.  I can't see any other way to make sure something like that doesn't happen.